catalog-info.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  annotations:
+    backstage.io/kubernetes-id: intellij-dependency-analytics
+    github.com/project-slug: RHEcosystemAppEng/intellij-dependency-analytics
+    github.com/project-readme-path: README.md
+    backstage.io/view-url: https://github.com/RHEcosystemAppEng/intellij-dependency-analytics/blob/main/catalog-info.yaml
```

This path is not correct
```suggestion
    backstage.io/view-url: https://github.com/redhat-developer/intellij-dependency-analytics/blob/main/catalog-info.yaml
```